poll/public/handlebars/poll_results.handlebars

```diff
@@ -1,13 +1,13 @@
 <script id="poll-results-template" type="text/html">
     <h3 class="poll-header">{{display_name}}</h3>
     <div class="poll-question-container">{{{question}}}</div>
-    <ul class="poll-answers-results poll-results">
+    <ul class="poll-answers-results poll-results {{~#if any_img}} has-images{{/if}}">
```

Comment:
@bradenmacdonald Could you explain to me what adding a `~` in front of the if statement signifies here?